usagov/usagov-2021
# USAGOV-1923-Popular-Services-Cutoff: Update core.entity_view_display.…

…node.directory_record.default.yml


## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1923

## Description

Changed Drupal Field Setting to not truncate text

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

- Go to any federal agency edit page
- Scroll down to "Top Tasks". Add a long string of text to the Link text field and any URL. Save and view. It will truncate to 80 characters. 
- Import new config
- Reload and it should no longer be truncated. 
- Go to any Spanish directory and again add a long string of text to the Link text field and any URL. Save and view. It should not be truncated at all. 
- 
## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
